src/app/navigation/examples/application-launcher-example.component.html

```diff
@@ -0,0 +1,61 @@
+<ul class="nav navbar-nav">
+</ul>
+<ul class="nav navbar-nav navbar-right navbar-iconic">
+    <li class="dropdown">
+        <pfng-application-launcher
+        [label] ="'Application Launcher test'"
+        [isDisabled] = "false"
+        [isList]="false"
+        [items]="navigationItems"
+        [hiddenIcons] = "false">
+      </pfng-application-launcher>
+    </li>
+    <li class="dropdown">
+      <a class="dropdown-toggle nav-item-iconic" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        <span title="Help" class="fa pficon-help"></span>
+        <span class="caret"></span>
+      </a>
+      <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+        <li><a href="#">Help</a></li>
+        <li><a href="#">About</a></li>
+      </ul>
+    </li>
+    <li class="dropdown">
+      <a class="dropdown-toggle nav-item-iconic" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        <span title="Username" class="fa pficon-user"></span>
+        <span class="caret"></span>
+      </a>
+      <ul class="dropdown-menu" aria-labelledby="dropdownMenu2">
+        <li><a href="#">Preferences</a></li>
+        <li><a href="#">Logout</a></li>
```

Comment:
These drop downs don't appear to function. For example, they don't open to show options. Is that intentional?